libs/dataset-explorer/src/lib/DatasetExplorerTab.tsx

```diff
@@ -87,18 +89,20 @@ export class DatasetExplorerTab extends React.PureComponent<
       return <div />;
     }
     const plotlyProps = generatePlotlyProps(
-      this.props.jointDataset,
+      this.context.jointDataset,
       this.state.chartProps,
-      this.props.cohorts[this.state.selectedCohortIndex]
+      this.context.cohorts.map((cohort) => cohort.cohort)[
```

Comment:
are these error cohorts instead of cohorts?  maybe change name to errorCohorts, then you won't have cohort.cohort.name like below which is a bit confusing but instead:
```
errorCohort.cohort.name
```

Reply:
I agree. This does look confusing. Will change!